contrib/ci/Jenkinsfile

```diff
@@ -1,27 +1,46 @@
+def lualint() {
+	sh 'luacheck package scripts targets'
+}
+
+def shelllint() {
+	echo "shellcheck"
```

Comment:
This will not do anything as you `echo` the string. Also shellcheck won't run without any parameters.

Reply:
I know its mark as WIP and it was a place holder for testing the parallel run. But I assume I have to rebuild the docker images? Because the jerkins did not show my new step?

Reply:
Jenkins will not execute changes to the Jenkinsfile in pull requests from non-members.